### PR TITLE
refactor: rename store action handlers

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-6891

### CODE REVIEW

File: src/config/config.ts [hunk 1]
```diff
@@ -41,8 +39,6 @@ export const defaultSettings: DefaultSettingsType = {
     'auto',
   languageMode: setLanguage() as LanguageKey,
   clientWallets: [],
-  activeWalletName:
-    (localStorage.getItem(localStorageKey.activeWalletName) as string) || '',
```
Comment (dennyscode) [inline src/config/config.ts:45]: We did not take advantage of that value and never used it! So, I got rid of it

File: src/stores/settings/SettingsStore.tsx [hunk 2]
```diff
@@ -35,28 +34,15 @@ export const useSettingsStore = createWithEqualityFn(
...
-          activeWalletName,
-        });
-      },
-
-      onWalletDisconnect: () => {
-        set({
-          activeWalletName: 'none',
-        });
-      },
-
```
Comment (dennyscode) [inline src/stores/settings/SettingsStore.tsx:50]: Same here, although we did set the wallet name once the wallet connection started + cleared the name once the wallet disconnects - it´s never been used.

So, those to handlers got removed as well.

**Review by DNR500 — APPROVED** [review]

**Review by 0xsebn — APPROVED** [review]
Good change, will have to update this on this on the scan as well.

**Review by tcheee — APPROVED** [review]
Ok, thank you for handling this!

**Pull request merged**